src/main/java/lotto/domain/Bonus.java

```diff
@@ -0,0 +1,33 @@
+package lotto.domain;
+
+import lotto.exception.ExceptionMessage;
+
+public class Bonus {
+    private final int bonusNumber;
+
+    private Bonus(int bonusNumber, Lotto lotto) {
+        validateRange(bonusNumber);
+        validateSameNumberInLotto(lotto, bonusNumber);
+        this.bonusNumber = bonusNumber;
+    }
+
+    public static Bonus of(int inputToNumber, Lotto lotto) {
+        return new Bonus(inputToNumber, lotto);
+    }
+
+    public boolean hasSameNumberInGeneratedLotto(Lotto generatedRandomLotto) { // 보너스 번호로 맞은 경우
+        return generatedRandomLotto.hasSameNumber(bonusNumber);
+    }
+
+    private void validateRange(int bonusNumber) {
+        if (bonusNumber > Lotto.MAX_VALUE || bonusNumber < Lotto.MIN_VALUE) {
```

Comment:
이렇게 if문 안에 다양한 연산자가 사용되는 상황에서, 직관적으로 이 검증문이 어떤 역할을 하는지 한 번 더 분기하면 좋을 것 같다는 의견입니다 :)
```suggestion
private boolean isInvalidNumber(int number){
        return bonusNumber > Lotto.MAX_VALUE || bonusNumber < Lotto.MIN_VALUE;
}

private void validateRange(int bonusNumber) {
        if (isInvalidRange(bonusNumber)) {
            throw new IllegalArgumentException(ExceptionMessage.OUT_OF_RANGE_BONUS_NUMBER.getErrorDescription());
        }
```

Reply:
가독성의 측면에서도 더 좋은 방법인 것 같아요!
연산자가 들어간 조건의 경우에는 메서드화 할게요~!